ZelBack/src/routes.js

```diff
@@ -1,912 +1,1770 @@
-    zelnodeService.zelfluxDebugLog(req, res);
+    fluxService.fluxDebugLog(req, res);
   });
   app.get('/zelnode/zelfluxinfolog', (req, res) => {
-    zelnodeService.zelfluxInfoLog(req, res);
+    fluxService.fluxInfoLog(req, res);
   });
   app.get('/zelnode/tailzelfluxerrorlog', (req, res) => {
-    zelnodeService.tailFluxErrorLog(req, res);
+    fluxService.tailFluxErrorLog(req, res);
   });
   app.get('/zelnode/tailzelfluxwarnlog', (req, res) => {
-    zelnodeService.tailFluxWarnLog(req, res);
+    fluxService.tailFluxWarnLog(req, res);
   });
   app.get('/zelnode/tailzelfluxdebuglog', (req, res) => {
-    zelnodeService.tailFluxDebugLog(req, res);
+    fluxService.tailFluxDebugLog(req, res);
   });
   app.get('/zelnode/tailzelfluxinfolog', (req, res) => {
-    zelnodeService.tailFluxInfoLog(req, res);
+    fluxService.tailFluxInfoLog(req, res);
   });
 
   app.get('/zelflux/broadcastmessage/:data?', (req, res) => {
-    zelfluxCommunication.broadcastMessageFromUser(req, res);
+    fluxCommunication.broadcastMessageFromUser(req, res);
   });
   app.get('/zelflux/broadcastmessagetooutgoing/:data?', (req, res) => {
-    zelfluxCommunication.broadcastMessageToOutgoingFromUser(req, res);
+    fluxCommunication.broadcastMessageToOutgoingFromUser(req, res);
   });
   app.get('/zelflux/broadcastmessagetoincoming/:data?', (req, res) => {
-    zelfluxCommunication.broadcastMessageToIncomingFromUser(req, res);
+    fluxCommunication.broadcastMessageToIncomingFromUser(req, res);
   });
   app.get('/zelflux/addpeer/:ip?', (req, res) => {
-    zelfluxCommunication.addPeer(req, res);
+    fluxCommunication.addPeer(req, res);
   });
   app.get('/zelflux/removepeer/:ip?', (req, res) => {
-    zelfluxCommunication.removePeer(req, res);
+    fluxCommunication.removePeer(req, res);
   });
   app.get('/zelflux/removeincomingpeer/:ip?', (req, res) => {
-    zelfluxCommunication.removeIncomingPeer(req, res, expressWs.getWss('/ws/zelflux'));
+    fluxCommunication.removeIncomingPeer(req, res, expressWs.getWss('/ws/zelflux'));
   });
   app.get('/zelflux/allowport/:port?', (req, res) => {
-    zelfluxCommunication.allowPortApi(req, res);
+    fluxCommunication.allowPortApi(req, res);
   });
   app.get('/zelflux/checkcommunication', (req, res) => {
-    zelfluxCommunication.isCommunicationEstablished(req, res);
+    fluxCommunication.isCommunicationEstablished(req, res);
   });
 
   app.get('/zelbench/start', (req, res) => {
-    zelnodeService.startZelBench(req, res);
+    fluxService.startBenchmark(req, res);
   });
   app.get('/zelbench/restart', (req, res) => {
-    zelnodeService.restartZelBench(req, res);
+    fluxService.restartBenchmark(req, res);
   });
   app.get('/zelbench/restartnodebenchmarks', (req, res) => {
-    zelbenchService.restartNodeBenchmarks(req, res);
-  });
-
-  app.get('/explorer/reindex/:reindexapps?', (req, res) => {
-    explorerService.reindexExplorer(req, res);
-  });
-  app.get('/explorer/restart', (req, res) => {
```

Comment:
wrongly deleted?

Reply:
its still there but more down.
APIs are now 'duplicatte' as of different path structure